db/init.sql

```diff
@@ -0,0 +1,94 @@
+-- This file initializes the SQL schema.
+--
+-- It is intended to serve as a migration that brings the database to the
+-- latest schema, too: when making changes, don't edit the existing code but
+-- rather add additional ALTER TABLE, CREATE TABLE, etc. statements to the
+-- bottom of this file and use IF NOT EXISTS to make it idempotent.
+
+-- ingest_runs stores one row for each time we run the ingester.
+CREATE TABLE IF NOT EXISTS ingest_run (
+  -- A unique ID for the ingestion run
+  ingest_id SERIAL PRIMARY KEY,
+  -- Timestamp the ingestion started at
+  ingest_dt TIMESTAMPTZ NOT NULL,
+  -- We provide 30-minute "waypoints" of historical data to the frontend. We
```

Comment:
For later thought -- Charlotte said 30 min but on election night itself I'm wondering if 5 min would be even better for the raw DB representation? then day after election night we'd probably want to return every-30-min waypoints, then day after that every-hour, and so on?